src/sprite.js

```diff
@@ -448,23 +449,7 @@ class Sprite {
    * @returns {Boolean|null} `true` if the objects collide, `false` otherwise. Will return `null` if the either of the two objects are rotated.
    */
   collidesWith(object) {
```

Comment:
We can go ahead and remove the function completely from `Sprite`, no need to call into the extracted function. 

Also if you could move the comment block to the `spriteCollidesWith` function that'd be super helpful (with updated example to show how to use it now. Don't need to copy anything past the `If you need a different type of collision check` since it no longer applies).

```js
 /**
   * Check if a sprite collides with an object. Uses a simple [Axis-Aligned Bounding Box (AABB) collision check](https://developer.mozilla.org/en-US/docs/Games/Techniques/2D_collision_detection#Axis-Aligned_Bounding_Box). Takes into account the sprites [anchor](api/sprite/#anchor).
   *
   * **NOTE:** Does not take into account sprite rotation. If you need collision detection between rotated sprites you will need to implement your own `collidesWith()` function. I suggest looking at the Separate Axis Theorem.
   *
   * ```js
   * import { Sprite, spriteCollidesWith } from 'kontra'; 
   *
   * let sprite = Sprite({
   *   x: 100,
   *   y: 200,
   *   width: 20,
   *   height: 40
   * });
   *
   * let sprite2 = Sprite({
   *   x: 150,
   *   y: 200,
   *   width: 20,
   *   height: 20
   * });
   *
   * spriteCollidesWith(sprite, sprite2);  //=> false
   *
   * sprite2.x = 115;
   *
   * spriteCollidesWith(sprite, sprite2);  //=> true
   * ```
   */
```